test/LogForwarding.test.js

```diff
@@ -19,6 +19,10 @@ const dataFixtures = [
     port: 'port1',
     index: 'index1',
     hec_token: 'token1'
+  }],
+  ['new_relic', 'setNewRelic', {
```

Comment:
This test is not necessary (as well as the method it covers). Those are written for deprecated methods. Maybe we even can get rid of those now?